src/beta/black-highlighter-beta.css

```diff
@@ -0,0 +1,11730 @@
+@charset "utf-8";
+
+@import url("https://cdn.jsdelivr.net/gh/Nu-SCPTheme/Black-Highlighter@BHL-Beta/src/fonts/fonts.css");
```

Comment:
Remember to change this when you merge.

Honestly I would have a list somewhere (maybe a comment on the PR) of all the things you need to fix up after merging.

Reply:
Good plan!

Reply:
Just switched this to the relative link, though the minified file will need to be changed to `@import "../../fonts/fonts.css";`